core-client/src/main/java/org/glassfish/jersey/client/internal/HttpUrlConnector.java

```diff
@@ -527,6 +536,20 @@ public Object run() throws NoSuchFieldException,
         }
     }
 
+    private void processExpect100Continue(ClientRequest request, HttpURLConnection uc, boolean allowStreaming) {
+
+        final Boolean expectContinueActivated = ClientProperties.getValue(request.getConfiguration().getProperties(),
```

Comment:
Use ClientRequest#resolveProperty

Reply:
for now this is being investigated because ClientRequest#resolveProperty does not resolve those properties which are put to clientConfiguration by the feature

Reply:
shall be fixed...